src/services/zapper.ts

```diff
@@ -325,7 +325,7 @@ export class ZapperService extends Service {
       toToken = ZeroAddress;
     }
 
-    const url = `https://api.zapper.fi/v1/zap-out/vault/${zapProtocol}/transaction`;
+    const url = `https://api.zapper.fi/v2/zap-out/vault/${zapProtocol}/transaction`;
```

Comment:
`shouldSellEntireBalance` seems a weird param ... not sure why that was added as it does not make much sense and is not documented. Should we remove it?

Reply:
I think so, removed with https://github.com/yearn/yearn-sdk/pull/284/commits/74168a7b58b2d38bba4e90718f9019e147aa3158